sections/tox.zsh

```diff
@@ -0,0 +1,54 @@
+#
+# Tox
+#
+# tox is a generic virtualenv management and test command line tool
+# Link: https://tox.readthedocs.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_TOX_SHOW="${SPACESHIP_TOX_SHOW=true}"
+SPACESHIP_TOX_PREFIX="${SPACESHIP_TOX_PREFIX=""}"
+SPACESHIP_TOX_SUFFIX="${SPACESHIP_TOX_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_TOX_SYMBOL="${SPACESHIP_TOX_SYMBOL="  "}"
+
+# ------------------------------------------------------------------------------
+# Section
+# ------------------------------------------------------------------------------
+
+spaceship_async_job_load_tox() {
+  [[ $SPACESHIP_TOX_SHOW == false ]] && return
+
+  async_job spaceship spaceship_async_job_tox "$PWD"
+}
+
+tox_test() {
+  # test code if tox config present
+  { command tox -q >/dev/null 2>&1 \
+    && { ret_code='OK' } \
+    || {  ret_code='FAIL' }}
+  echo $ret_code
+}
+
+spaceship_async_job_tox() {
+  builtin cd -q "$1" 2>/dev/null
+  # test directly if tox.ini present in current folder...
+  { [[ -f tox.ini ]] && tox_test } || \
+    # ...or try tox return, little longer
+    { command tox --notest 2>&1 | grep -q 'ERROR: toxini' && return "" || tox_test }
+}
```

Comment:
Why not make this function this simple?

```
spaceship_async_job_tox() {
  builtin cd -q "$1" 2>/dev/null
  [[ -f tox.ini ]] || return
  tox -q >/dev/null 2>&1 && echo 'OK' || echo 'FAIL'
}
```

Reply:
I do this because in a project, I would like to have test running even if i'm in subfolder of a project.
I find tox little long to find there is nothing to do so I have done a short-circuit...but yes, this make code more complicated...